lib/channel.rb

```diff
@@ -0,0 +1,50 @@
+require_relative "recipient"
+require "pry"
+require "dotenv"
+require "httparty"
+require "table_print"
+
+Dotenv.load
+
+module Slack
+  class Channel < Recipient
+    attr_reader :topic, :member_count
+
+    def initialize(slack_id:, name:, topic:, member_count:)
+      super(slack_id, name)
+      @topic = topic
+      @member_count = member_count
+    end
+
+    BASE_URL = "https://slack.com/api/"
+    CHANNEL_URL = "https://slack.com/api/channels.list"
+
+    def details
+      return {
+        Name: @name.capitalize,
+        Slack_id: @slack_id,
+        Topic: @topic,
+        Member_count: @member_count
+      }
+    end
+
+    def self.channels_get
+      query_params = { token: ENV["SLACK_API_TOKEN"] }
+      return Slack::Channel.get(CHANNEL_URL, query_params)
+    end
```

Comment:
Since you're already inside of `Slack::Channel`, you can just say `return get(...)` on line 33.